runtime/launcher.m

```diff
@@ -581,7 +581,19 @@ int xamarin_main (int argc, char **argv, bool is_extension)
 		*ptr = NULL;
 
 		if (is_extension) {
-			rv = NSExtensionMain (new_argc, new_argv);
+			void * libExtensionHandle = dlopen ("/usr/lib/libextension.dylib", RTLD_LAZY);
```

Comment:
who's responsible for installing that dylib the `/usr/lib/` on customer's Mac ? should not it be part of the app bundle ? (e.g. versioning issues)